### ISSUE TITLE
Code samples for SlackAttachment

### ISSUE THREAD

**Issue by adriens** [gpedro/slack-webhook#27]
Hi,

still me...could you please provide some code samples for ```SlackAttachment``` to see how to format nice looking messages ?

**Comment by adriens** [comment]
All done, it looks great !...worked like a charm, made the mapping between your API and slack guidelines at https://api.slack.com/docs/message-attachments to better understanf what each field means

**Comment by gpedro** [comment]
i think should better add a wiki sections explaining more details about your api. what do you think? @adriens

**Comment by adriens** [comment]
Yep, Wiki or a new markdown on the project, referenced by the README.md ... 'cause i love to read markdown from the welcome page, i find that very convenient ;-p ... but it's just my personal preference <3

**Comment by adriens** [comment]
...an another approach would to complete javadoc qui some dummy code sample

**Comment by adriens** [comment]
...and to finish...and another idea would to reference projects that use your project so real code samples could be seen.

As soon as i've published my little project, i'll let you know and quote your project anyway :smile_cat:

**Comment by gpedro** [comment]
javadoc and extended readme looks great to me

**Comment by adriens** [comment]
Yep, for me too :+1:

**Comment by gpedro** [comment]
can you help me with javadoc?

**Comment by adriens** [comment]
Sure, i'll can make you some PR as soon as i will have publushed the BOT. As i'm doing this on evenings and WEs, stuff is not going so fast... but you already probably know the deal about that too :smile_cat: , don't you ?

**Comment by adriens** [comment]
The ```1.0``` of my project is about to be released.
Remains me to write a little linkedIn article and i'll can help you with javadoc 👍

**Comment by adriens** [comment]
Article written @gpedro , see https://www.linkedin.com/pulse/birthdays-bots-slack-travis-ci-cron-jobs-all-free-fun-adrien-sales/!
